source/index.html.md.erb

```diff
@@ -7680,9 +7714,11 @@ Extends the [Row Object](#row-object), adding the following:
 
      |     |     |
 -----|-----|-----|
+**readOnlyFullAccessibleBy** | string | Flag to indicate who can access the 'Read-Only Full' view of the published report:<ul><li>If "ALL", it is available to anyone who has the link.</li><li>If "ORG", it is available only to members of the report owner's Smartsheet organization.</li></ul>Only returned in a response if **readOnlyFullEnabled** = true.
+**readOnlyFullDefaultView** | enum | Indicates which view the user has set for a read-only, default view of the published report. 0 or null = GRID, 1 = CARD, and 2 = CALENDAR.
 **readOnlyFullEnabled** | Boolean | If `true`, a rich version of the report is published with the ability to download row attachments and discussions.
+**readOnlyFullShowToolbar** | Boolean | If `true`, shows the left nav toolbar.
```

Comment:
True is the default. So if false - it is hidden

Reply:
I clarified it.